### DISCUSSION TITLE
[client] Sort comments by age, votes, ...

### DISCUSSION THREAD

**Discussion by rawsh** [posativ/isso#792] [Q&A] [answered]
Title

**Comment by blatinier** [comment]
Is this an error? Should I close? If not please provide more information. Sort what, on what, where…

**Comment by rawsh** [comment]
How can we sort comments by new, votes, old, etc. @blatinier 
Nothing in docs.

**Comment by blatinier** [comment]
Nothing available so far to do it.
It's mentioned here: #4  and there's a PR (which fails at implementing it completly) here: #322

**Comment by rawsh** [comment, marked as answer]
@blatinier client side:

```
#isso-root {
    display: flex;
    flex-direction: column-reverse;
}
.isso-comment {
    flex: 0;
    width: 100%;
}
```

e.g. https://www.devpy.me/10-of-the-best-linux-icon-themes-compared/#isso-thread

**Comment by budui** [comment]
It may be useful and easy to add a url parma to [fetch api](https://github.com/posativ/isso/blob/dfbc80d5b645f0754de6884078db29ac6a6ad642/isso/views/comments.py#L732), because there are some parma(asc, order_by) in [db fetch api](https://github.com/posativ/isso/blob/dfbc80d5b645f0754de6884078db29ac6a6ad642/isso/db/comments.py#L199) for the order of comments.

**Comment by rawsh** [comment]
Yeah that would be a clean solution. It could be a client option then in the js

**Comment by ix5** [comment]
@budui @rawsh do you feel like working on this?

**Comment by ix5** [comment]
#4 already addresses this, closing as dupe.
